linux_os/guide/services/ssh/ssh_server/sshd_use_approved_ciphers/oval/shared.xml

```diff
@@ -1,45 +1 @@
-<def-group>
-  <definition class="compliance" id="sshd_use_approved_ciphers" version="1">
-    <metadata>
-      <title>Use Only Approved Ciphers</title>
-      <affected family="unix">
-        <platform>multi_platform_wrlinux</platform>
```

Comment:
Please review the prodtype from `rule.yml` according to affected platform. Here I can see that is multi platform wrlinux but in `prodtype` it has only `prodtype: wrlinux1019,rhel6,rhel7,rhel8,ol7,rhv4`. Missing the wrlinux8.

Reply:
Actually, I'm not sure it was the intention to restrict to `wrlinux1019`. We need to investigate.

Reply:
I will take a look and compare oval platforms and `rule.yml` platforms. It's possible that someone changed oval platforms but forgot about `rule.yml`.

Reply:
I checked sshd rules and for most of them platforms in ovals don't correspond to `rule.yml` prodtypes and it's not possible to visit complete history of `rule.yml`. In some cases `prodtype` in `rule.yml` is missing (so it includes all products), but in OVAL it doesn't include all products and vice versa (e.g. example you wrote).

I will include to `rule.yml` products, which were in OVAL but not in `prodtype`. But I won't restrict `prodtype` if it includes products, which were not mentioned in OVAL `platform`.

Reply:
Ok. I'm okay with that.